### ISSUE TITLE
How do you think the future of UI development looks like?

### ISSUE THREAD

**Issue by AronParker** [shanselman/ama#72]
First of all, I really appreciate your character. I've seen the Head to Head on NDC conferences with you and Jon Skeet and it was hilarious, you've got a great sense of humor and a great attitude!

What do you think about the future of UI development? There are several frameworks out there, many of which seem to be in decline like MFC, WinForms, WPF, Swing, awt, etc. where UWP, Xamarin, JavaFX, Qt seem to be on the rise. Which ones do you think will be valuable? Or will everything be moved to web via HTML5, css & frameworks so that it's easily available on all devices?

So in short, what do you think is going to be used in the future of Desktop user interface development?
